src/dev/impl/DevToys/Views/Tools/EncodersDecoders/CertificateEncoderDecoder/CertificateEncoderDecoderToolPage.xaml

```diff
@@ -0,0 +1,86 @@
+﻿<Page
+    x:Class="DevToys.Views.Tools.CertificateEncoderDecoder.CertificateEncoderDecoderToolPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:DevToys.UI.Controls"
+    xmlns:toolkit="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    mc:Ignorable="d"
+    NavigationCacheMode="Required">
+
+    <Grid x:Name="RootGrid" RowSpacing="12" Margin="0,0,0,16">
+        <Grid.ChildrenTransitions>
+            <TransitionCollection>
+                <EntranceThemeTransition IsStaggeringEnabled="True" FromVerticalOffset="50"/>
+            </TransitionCollection>
+        </Grid.ChildrenTransitions>
+
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition MinWidth="100"/>
+            <ColumnDefinition Width="16"/>
+            <ColumnDefinition MinWidth="100"/>
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition MinHeight="100"/>
```

Comment:
I suspect these MinHeight are unnecessary since we already set a (different) MinHeight in the control itself on line 79.

Reply:
You're right.  I had to leave the second one:
```xaml
<Grid.RowDefinitions>
    <RowDefinition Height="Auto"/>
    <RowDefinition Height="Auto"/>
    <RowDefinition Height="Auto"/>
    <RowDefinition MinHeight="100"/>
</Grid.RowDefinitions>
```

Because when it was set like this, the controls were not extending to the bottom of the UI:
```xaml
<Grid.RowDefinitions>
    <RowDefinition Height="Auto"/>
    <RowDefinition Height="Auto"/>
    <RowDefinition Height="Auto"/>
    <RowDefinition Height="Auto"/>
</Grid.RowDefinitions>
```
![image](https://user-images.githubusercontent.com/74258557/234693575-a9426389-0302-4dff-b958-ab0fb9e89793.png)